### PR TITLE
Adding some more metric counterz

### PR DESCRIPTION
**Pull request by matzew**
Adding some more push metrics, per platform/batch job

See:
https://issues.jboss.org/browse/AGPUSH-2225

### CODE REVIEW

**Comment by pb82** [comment]
👀

File: push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/AdmPushNotificationSender.java [hunk 1]
```diff
@@ -34,6 +35,11 @@
 public class AdmPushNotificationSender implements PushNotificationSender {
     private final Logger logger = LoggerFactory.getLogger(AdmPushNotificationSender.class);
 
+    private static final Counter promPrushRequestsAdm = Counter.build()
+            .name("aerogear_ups_push_requests_adm")
```
Comment (pb82) [inline push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/AdmPushNotificationSender.java]: What does the `adm` here stand for? Admin? Is there a way to trigger a batch requests in the UI?

File: push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/WNSPushNotificationSender.java [hunk 2]
```diff
@@ -58,6 +59,11 @@
 
     private final Logger logger = LoggerFactory.getLogger(WNSPushNotificationSender.class);
 
+    private static final Counter promPrushRequestsWindows = Counter.build()
+            .name("aerogear_ups_push_requests_windows_wns")
+            .help("Total number of Android push batch requests.")
```
Comment (pb82) [inline push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/WNSPushNotificationSender.java]: Android -> Windows

**Comment by pb82** [comment]
@matzew I sent push messages to Android and iOS Variants but only the total counter (aerogear_ups_push_requests_total) was increased. The other counters don't appear in the metrics output (aerogear_ups_push_requests_ios, aerogear_ups_push_requests_android).

File: push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/AdmPushNotificationSender.java (hunk 1 above)
Comment (matzew) [reply push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/AdmPushNotificationSender.java]: adm -> Amazon device messaging (the class is also named ADMpush sender)

**Comment by matzew** [comment]
> I sent push messages to Android and iOS Variants but only the total counter...

did you register devices ? If not - see in the code, on empty token collection, it immediately returns ;-)

**Pull request merged**